cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -153,6 +153,7 @@ function Autocomplete( element, opts ) {
 
   /**
    * Event handler for input into the _input element
+   * @param {Object} event The event object
```

Comment:
You could log this to confirm, but pretty sure the type here would be [`InputEvent`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) not `Object`.